slaveapi/actions/reboot.py

```diff
@@ -74,7 +74,7 @@ def reboot(name, update_bug=True):
         status_text += "Attempting PDU reboot..."
         try:
             try:
-                slave.pdu.powercycle()
+                slave.pdu.reboot()
```

Comment:
As i said in bug, I'm less happy about a .reboot() compared to what we've seen in the past with machines. (e.g. windows, where the ssh is hung but ping stays up -- and then we don't actually notice the machine's network interface go down and think its a broken host).

BUUUT If you (buildduty) and Amy like this approach better I'm all for going with it.